run_tests

```diff
@@ -13,7 +13,7 @@ else
 fi
 
 flake8 || exit $?
-black --diff --check --skip-string-normalization src tests || exit $?
+black --diff --check src tests || exit $?
 which pytest > /dev/null
```

Comment:
Are we going to use pytest here?

Reply:
It uses pytest if installed

Reply:
@dstathis I was wondering if shouldn't we run: `black -l 79 --diff --check src tests || exit $?` so we would comply with [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)

Reply:
Flake8 runs on the previous line and will catch that

Reply:
@jlounder Do we have a Canonical standard for line length?